SpatialConstAffine.lua

```diff
@@ -0,0 +1,39 @@
+local SpatialConstAffine, parent = torch.class('inn.SpatialConstAffine', 'nn.Module')
+
+function SpatialConstAffine:__init(nOutputPlane, inplace)
+  parent.__init(self)
+  self.a = torch.Tensor(1,nOutputPlane,1,1)
```

Comment:
Well, I think something should be done in here, it's not a trainable module, and `a` and `b` are initialized with garbage. maybe a function to set `a` and `b`, or pass `a` and `b` in the constructor...
You know what, if you pass `a` and `b` as arguments, you can have a generic `ConstAffine`, in whichever dimension you want.